### PR TITLE
Upkeep for Playground deployment

### PR DESCRIPTION
**Pull request by shepmaster**
These changes have been running in production for a while after I hand-applied them. The Ansible-specific changes are mostly to see that the managed changes match close enough to the manual changes, but hopefully will be helpful to others.

### CODE REVIEW

File: ansible/playbooks/playground.yml [hunk 1]
```diff
@@ -20,9 +20,10 @@
         - "{{ vars_playground_domain }}"
 
     - role: nginx
+      worker_connections: "{{ vars_playground_number_connections }}"
       proxied:
         - domain: "{{ vars_playground_domain }}"
-          to: "http://localhost:{{ vars_playground_env_ui_port }}"
+          to: "http://127.0.0.1:{{ vars_playground_env_ui_port }}"
```
[the remark this answers is not in the document]
Comment (shepmaster) [reply ansible/playbooks/playground.yml:26]: Check out [the commit message](https://github.com/rust-lang/simpleinfra/pull/311/commits/505f4f9585ebae95fee40a4f49de6bea4add11b6)

File: ansible/roles/nginx/templates/override.conf [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[Service]
+LimitNOFILE={{ (worker_connections * 2) + 32 }}
```
[the remark this answers is not in the document]
Comment (shepmaster) [reply ansible/roles/nginx/templates/override.conf:6]: Comment added

**Review by jdno — APPROVED** [review]
Looks good to me from the outside. The remark about FQDN is valid, but something we should fix for all Ansible roles at once.

**Comment by shepmaster** [comment]
Rebased now that #393 is merged and I added the last commit (the container GC script)

**Review by MarcoIeni — APPROVED** [review]

**Pull request merged**